website/blog/2025-06-12-react-native-0.80.md

```diff
@@ -0,0 +1,225 @@
+
+We experimented and benchmarked how much time this initial prebuild for iOS is saving and, in our benchmarks, run on an M4 machine, iOS builds are roughly 12% faster with the prebuild rather than by building from source.
+
+From our experience, we also observed that several bug reports from users are caused by build issues related with React Native’s 3rd party dependencies (example [#39568](https://github.com/facebook/react-native/issues/39568)).
+Prebuilding the 3rd party dependencies allows us to build them for you, so that you won’t face these build issues anymore.
+
+Note that we are not pre-building the whole React Native: we are only pre-building the libraries Meta does not directly control, such as Folly and GLog.
+
+In a future release, we will also ship the rest of React Native core as a prebuild.
+
+#### How to use them
+
+This feature is still experimental, so it is not turned on by default.
+
+If you want to use them, you can install your pods by adding the `RCT_USE_RN_DEP` env variable:
+
+```bash
+RCT_USE_RN_DEP=1 bundle exec pod install
+```
+
+Alternatively, if you want to enable it for all the developers working on that, you can modify your Podfile like this:
+
+```diff
+if linkage != nil
+  Pod::UI.puts "Configuring Pod with #{linkage}ally linked Frameworks".green
+  use_frameworks! :linkage => linkage.to_sym
+end
+
++ENV[‘`RCT_USE_RN_DEP`’] = ‘1’
+
+target 'HelloWorld' do
+  config = use_native_modules!
+```
+
+Please, report any issue that the prebuilds might cause to you and to your app in [this discussion](https://github.com/react-native-community/discussions-and-proposals/discussions/912). We are committed to looking into them and making sure that tier usage is transparent to your app.
+
+## Other Changes
+
+### Android - Smaller APK size thanks to IPO
+
+This release ships with significant size reduction for all the Android apps built with React Native. Starting in 0.80, we enabled [Interprocedural Optimization](https://en.wikipedia.org/wiki/Interprocedural_optimization) for both React Native and Hermes builds.
+
+This resulted in a saving of ~1Mb for all the Android apps.
+
+![android apk size comparison](../static/blog/assets/0.80-android-apk-size.png)
+
+You will get this size win by updating your React Native version to 0.80 and there are no further changes required to your project.
+
+### New App Screen redesign
+
+If you’re not using Expo but you’re using the Community CLI & Template, in this version we've moved the New App Screen into its [own package](https://www.npmjs.com/package/@react-native/new-app-screen) and given it a fresh coat of paint. This reduces initial code boilerplate when you create a new app with the Community Template, and also provides a better experience when viewed on larger screens.
+
+![New App Screen redesign](../static/blog/assets/0.80-new-community-template-landing.png)
```

Comment:
@cortinico Slightly fixed up + optimised version of this asset. Please download from https://pxl.cl/7s0Hn.

Reply:
updated